src/Microsoft.PowerShell.Commands.Utility/commands/utility/GetUptime.cs

```diff
@@ -0,0 +1,82 @@
+/********************************************************************++
+Copyright (c) Microsoft Corporation.  All rights reserved.
+--********************************************************************/
+
+using System;
+using System.Management.Automation;
+using System.Diagnostics;
+
+namespace Microsoft.PowerShell.Commands
+{
+    /// <summary>
+    /// This class implements Get-Uptime.
+    /// </summary>
+    [Cmdlet(VerbsCommon.Get, "Uptime", DefaultParameterSetName = "Timespan", HelpUri = "")]
+    public class GetUptimeCommand : PSCmdlet
+    {
+        /// <summary>
+        /// Timespan parameter
+        /// Time since the system started up
+        /// </summary>
+        /// <value></value>
+        [Parameter(ParameterSetName = "Timespan")]
+        public SwitchParameter Timespan { get; set; } = new SwitchParameter();
+
+        /// <summary>
+        /// Since parameter
+        /// The system startup time
+        /// </summary>
+        /// <value></value>
+        [Parameter(ParameterSetName = "Since")]
+        public SwitchParameter Since { get; set; } = new SwitchParameter();
+
+        /// <summary>
+        /// Pretty parameter
+        /// Convert output to default formated string
+        /// </summary>
+        /// <value></value>
+        [Parameter(ParameterSetName = "Timespan")]
+        [Parameter(ParameterSetName = "Since")]
+        public SwitchParameter Pretty { get; set; } = new SwitchParameter();
+
+        /// <summary>
+        /// ProcessRecord() override.
+        /// This is the main entry point for the cmdlet.
+        /// </summary>
+        protected override void ProcessRecord()
+        {
+            // Get-Uptime return null if IsHighResolution = false  
+            // because stopwatch.GetTimestamp() return DateTime.UtcNow.Ticks
+            // instead of ticks from system startup
```

Comment:
I don't know if it's possible to overload `Stopwatch.IsHighResolution`, especially in the case of binary modules where powershell doesn't play a role of resolving types.
Test hooks may help you achieve this, take a look at [how it's used in Get-Help -Online](https://github.com/PowerShell/PowerShell/blob/master/src/System.Management.Automation/help/HelpCommands.cs#L623), and [its test](https://github.com/PowerShell/PowerShell/blob/master/test/powershell/engine/HelpSystem.OnlineHelp.Tests.ps1#L28).

However, I think using a test hook here may be an overkill -- we only throw a terminating error in this case and *maybe* a one-time manual check would be enough. @JamesWTruher, can you please weigh in?